app/controllers/providers_controller.rb

```diff
@@ -63,6 +63,12 @@ def publish
   def training_providers
     if user_is_admin?
```

Comment:
Do we need to rebase with master to remove this?

Reply:
I did but looks like this conditional wasn't removed as a part of https://github.com/DFE-Digital/publish-teacher-training/pull/998 I will add another commit and remove it.